code/game/objects/items/powerfist.dm

```diff
@@ -11,10 +13,10 @@
 	attack_weight = 1
 	throwforce = 10
 	throw_range = 7
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_BULKY
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 40, "stamina" = 0)
 	resistance_flags = FIRE_PROOF
-	var/click_delay = 1.5
+	var/click_delay = POWERFIST_ATTACK_DELAY_MULT
```

Comment:
What I meant here is that you don't need the variable at all.
You can just put that define right into every place that click_delay is used